backfill/2024-07-15-fenix_active_users_aggregates_v3/fenix_query.sql

```diff
@@ -0,0 +1,123 @@
+WITH distribution_id AS
+(
+  SELECT
+    client_info.client_id,
+    ARRAY_AGG(
+      metrics.string.metrics_distribution_id IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS distribution_id
+  FROM
+    `moz-fx-data-shared-prod.fenix.metrics` -- from 2021 until 2023-12-31
+    -- `moz-fx-data-shared-prod.fenix.baseline` -- from 2024-01-01
+  WHERE
+    DATE(submission_timestamp) BETWEEN DATE_SUB(@submission_date, INTERVAL 28 DAY) AND DATE_ADD(@submission_date, INTERVAL 7 DAY)
```

Comment:
This is the relevant change, covering the previous 28 days.

Reply:
Should this be the previous 28 + 7 = 35 days? If I recall correctly, we chose +/- 7 days from the submission date in order to capture as much of the relevant pings as possible. So I think it could also make sense to include 7 days before the 28-day window as well.

Are there any reasons not to extend the window to 35 days?

Reply:
I don't see a reason not to, so I updated it with a note for now. I'll rerun my pre-validation tomorrow to make sure this doesn't impact.

Reply:
Running the query for 28 / 35 days for some sample dates doesn't change the results for baseline ping -which is expected because the ping will be there and the value doesn't change over time so 28 days should be enough in this case. However I'll leave it for alignment in the logic and for possible edge cases.